code/game/objects/items/cards_ids.dm

```diff
@@ -685,6 +688,44 @@ update_label("John Doe", "Clowny")
 	hud_state = JOB_HUD_RAWCARGO
 	access = list(ACCESS_MINING, ACCESS_MINING_STATION, ACCESS_MECH_MINING, ACCESS_MAILSORTING, ACCESS_MINERAL_STOREROOM)
 
+/obj/item/card/id/paper
+	name = "paper nametag"
+	desc = "Some spare papers taped into a vague card shape, with a name scribbled on it. Seems trustworthy."
+	icon_state = "paper"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 50, "stamina" = 0)
```

Comment:
Use the DEFINES here `MELEE`, etc

Reply:
literally every armor define uses this exact syntax i just copy pasted?

Reply:
there are defines, a lot of armor is out of date

Reply:
so....where is this and how do i define it

Reply:
the defines already exist, literally just change the strings to `MELEE` etc